src/main/webapp/app/components/oncokbTable/OncoKBTable.tsx

```diff
@@ -50,17 +51,34 @@ export default class OncoKBTable<T> extends React.Component<
   render() {
     return (
       <div>
-        {this.props.disableSearch ? undefined : (
-          <div className="d-flex">
-            <div className="ml-auto">
-              <input
-                onChange={(event: any) => {
-                  this.searchKeyword = event.target.value.toLowerCase();
-                }}
-                className="form-control input-sm"
-                type="text"
-                placeholder="Search ..."
-              />
+        {this.props.filters === undefined && this.props.disableSearch ? (
+          <></>
+        ) : (
+          <div className="mt-2 row">
```

Comment:
@zhx828 the cause of the screenshot test failure is here I believe. Since a margined row is created regardless of whether there are filters, there will be a few extra pixels above the search bar on other tables without filters. The old logic was that the margined row was only created if the filters were present, but I believe we decided to change it to creating a row regardless and adding filters and the search bar after the row is created.